openwisp_notifications/tests/test_transaction_notifications.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from django.core.cache import cache
+from django.db.models.signals import post_save
+from django.test import TransactionTestCase
+
+from openwisp_notifications.handlers import register_notification_cache_update
+from openwisp_notifications.signals import notify
+from openwisp_notifications.swapper import load_model
+from openwisp_users.tests.utils import TestOrganizationMixin
+
+User = get_user_model()
+Notification = load_model('Notification')
+
+
+class TestTransactionNotifications(TestOrganizationMixin, TransactionTestCase):
```

Comment:
We don't need to create another file for this. You can move this class to `test_notifications.py`. I believe it will make more sense there.